pkg/gds/emails/client_test.go

```diff
@@ -114,11 +114,13 @@ func TestClientSendEmails(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 1, sent)
 
-	// TODO: For reissuance related emails, test that emails are not sent twice
```

Comment:
Note that these tests are skipped by default because they actually use SendGrid to send the emails. I did set some local configuration and made sure that they pass with the changes.